### PR TITLE
Revert to the old version as Apple Developer Technical Support advices.

### PR DESCRIPTION
**Pull request by kanottonp**

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -1,22 +0,0 @@
-// swift-tools-version:5.0
-
```
Comment (chalermpong) [inline Package.swift:2]: This file should not be removed

**Review by chalermpong — COMMENTED** [review]
Package.swift should not be removed

**Review by chalermpong — APPROVED** [review]

File: SwipePopViewController.podspec [hunk 2]
```diff
@@ -10,15 +10,32 @@ Pod::Spec.new do |s|
   s.name             = 'SwipePopViewController'
   s.version          = '0.2.3'
   s.summary          = 'Swipe right to pop the current View Controller'
+
```
Comment (chalermpong) [inline SwipePopViewController.podspec:13]: This file should not be changed

**Review by chalermpong — COMMENTED** [review]
There should be no change in podspec

**Review by chalermpong — APPROVED** [review]

**Review by camp191 — APPROVED** [review]

**Pull request merged**